.github/actions/run-ee-server/action.yml

```diff
@@ -1,81 +1,69 @@
+  server-type:
     required: true
-    description: Specify Docker tag
-    default: 'latest'
+    default: aerospike-server-enterprise
+    description: Server type `aerospike-server` or `aerospike-server-enterprise`
+  strong-consistency:
+    required: false
+    default: "1"
+    description: Enable strong consistency (1 or 0)
+  security:
+    required: false
+    default: "0"
+    description: Enable security (1 or 0)
 
 runs:
-  using: "composite"
+  using: composite
   steps:
-  - name: Log into registry to get non-public server RCs
-    # We can still pull public images while logged in, so just do this all the time to make things simple
-    uses: docker/login-action@v3
+  - name: Set up JFrog credentials
+    id: setup-jfrog-cli
+    uses: jfrog/setup-jfrog-cli@5b06f730cc5a6f55d78b30753f8583454b08c0aa # v4.8.1
+    env:
+      JF_URL: ${{ inputs.jfrog-platform-url }}
+      JF_PROJECT: database
     with:
-      registry: ${{ inputs.registry-name }}
-      username: ${{ inputs.registry-username }}
-      password: ${{ inputs.registry-password }}
+      oidc-provider-name: ${{ inputs.oidc-provider }}
+      oidc-audience: ${{ inputs.oidc-audience }}
 
-  - run: echo IMAGE_FULL_NAME=${{ inputs.registry-name }}/${{ inputs.image-name }}:${{ inputs.server-tag }} >> $GITHUB_ENV
+  - run: docker login ${{ inputs.container-repo-url }} --username ${{ steps.setup-jfrog-cli.outputs.oidc-user }} --password ${{ steps.setup-jfrog-cli.outputs.oidc-token }}
     shell: bash
 
-  # macOS Github runners and Windows self-hosted runners don't have buildx installed by default
-  - if: ${{ runner.os == 'Windows' || runner.os == 'macOS' }}
-    uses: docker/setup-buildx-action@v3
-
-  - name: Build and push
-    id: docker-build
-    uses: docker/build-push-action@v6
-    with:
-      # Don't want to use default Git context or else it will clone the whole client repo again
-      context: .github/workflows/docker-build-context
-      build-args: |
-        server_image=${{ env.IMAGE_FULL_NAME }}
-      # setup-buildx-action configures Docker to use the docker-container build driver
-      # This driver doesn't publish an image locally by default
-      # so we have to manually enable it
-      load: true
-
-  - run: echo SERVER_CONTAINER_NAME="aerospike" >> $GITHUB_ENV
-    shell: bash
-
-  - run: docker run -d --name ${{ env.SERVER_CONTAINER_NAME }} -e DEFAULT_TTL=2592000 -p 3000:3000 ${{ steps.docker-build.outputs.digest }}
-    shell: bash
-
-  - uses: ./.github/actions/wait-for-as-server-to-start
-    with:
-      container-name: ${{ env.SERVER_CONTAINER_NAME }}
-      is-security-enabled: false
-      is-strong-consistency-enabled: true
-
-  # All the partitions are assumed to be dead when reusing a roster file
-  - run: docker exec ${{ env.SERVER_CONTAINER_NAME }} asadm --enable --execute "manage revive ns test"
+  - run: echo IMAGE_FULL_NAME=artifact.aerospike.io/oci/${{ inputs.server-type }}:${{ inputs.server-tag }} >> $GITHUB_ENV
```

Comment:
We still have that OCI image part that we need to work through (on the platform side) so this path will likely have to change